index.ts

```diff
@@ -163,48 +181,52 @@ export function autocomplete ({
  * @param  {string} [$0.url]
  * @return {Promise}                A Promise that'll get resolved with the bulk geocode result
  */
-export function bulk ({
+export function bulk({
   addresses,
+  boundary,
   clientId,
   clientSecret,
-  boundary,
   focusPoint,
   url
 }: BaseQuery & {
-  addresses: Array<any>,
-  boundary?: Boundary,
-  focusPoint?: any
-}): Promise<BaseResponse> {
-  const geocoder = getGeocoder(clientId, clientSecret, url)
-  let addressesQuery = [...addresses]
-  const options = {}
+  addresses: Array<string>
+  boundary?: Boundary
+  focusPoint?: Point
+  text?: string
+}): Promise<GeoJSON.FeatureCollection> {
```

Comment:
This is missing the query part of the response. See recommended change:

```suggestion
}): Promise<GeoJSON.FeatureCollection & { query: { addresses: String[] } }> {
```

Reply:
Refactored in aecf758d679984328a465df451fd953bde17efbf